lenscalc_web/app.py

```diff
@@ -48,6 +48,6 @@ def result(variables=None, values=None):
     except ValueError:
         return flask.render_template("index.html", variables=variables_info)
 
-    calculated_values = lens.parameters
+    calculated_values = lens.__dict__
```

Comment:
Be careful here. `__dict__` might contain also extra attributes.

Reply:
This is in the web app, so from my understanding no one should be able to add an extra attribute.
[This](https://github.com/adelpopelkova/lenscalc/blob/0f2a6af2cadf17bd3e64badd72470c55e95ddb84/lenscalc_web/templates/result.html#L51) is where the dictionary is used so extra attributes should not affect the result.